### PR TITLE
Fix various typos

### PR DESCRIPTION
**Pull request by luzpaz**
Found via `codespell -q 3 -L aci,ake,aline,alle,alledges,alocation,als,ang,anid,apoints,ba,beginn,behaviour,bloaded,bottome,byteorder,calculater,cancelled,cancelling,cas,cascade,centimetre,childrens,childs,colour,colours,commen,connexion,currenty,dof,doubleclick,dum,eiter,elemente,ende,feld,finde,findf,freez,hist,iff,indicies,initialisation,initialise,initialised,initialises,initialisiert,inout,ist,kilometre,lod,mantatory,methode,metres,millimetre,modell,nd,noe,normale,normaly,nto,numer,oce,oder,ontop,orgin,orginx,orginy,ot,pard,parm,parms,pres,programm,que,rady,recurrance,rougly,seperator,serie,sinc,strack,substraction,te,thist,thru,tread,uint,unter,vertexes,wallthickness,whitespaces -S ./.git,*.po,*.ts,./ChangeLog.txt,./src/3rdParty,./src/Mod/Assembly/App/opendcm,./src/CXX,./src/zipios++,./src/Base/swig*,./src/Mod/Robot/App/kdl_cp,./src/Mod/Import/App/SCL,./src/WindowsInstaller,./src/Doc/FreeCAD.uml,./build/doc/SourceDocu`

### CODE REVIEW

**Comment by luzpaz** [comment]
@WandererFan please review c8080eb specifically. Thanks!

File: src/Mod/Assembly/App/opendcm/core/property.hpp [hunk 1]
```diff
@@ -74,7 +74,7 @@ namespace dcm {
  * A property of kind @ref vertex_property will added to vertices, a property of kind @ref object_property to all
  * objects and so on.
  *
- * If the property type is a standart c++ type like int or bool, the defualt value can't be set by using its
+ * If the property type is a standart c++ type like int or bool, the default value can't be set by using its
```
Comment (chennes) [inline src/Mod/Assembly/App/opendcm/core/property.hpp:77]:
```suggestion
 * If the property type is a standard c++ type like int or bool, the default value can't be set by using its
```

**Comment by chennes** [comment]
Do you want me to hold this for @WandererFan's review?

**Comment by luzpaz** [comment]
@chennes, might as well. No rush :wink:

**Comment by freecadci** [comment]
<a href="https://gitlab.com/freecad/FreeCAD-CI/-/commits/PR_5374">pipeline status</a> for feature branch [PR_5374](https://gitlab.com/freecad/FreeCAD-CI/-/commits/PR_5374). Pipeline [446454464](https://gitlab.com/freecad/FreeCAD-CI/-/pipelines/446454464) was triggered at [1508f9a](https://github.com/FreeCAD/FreeCAD/pull/5374/commits/1508f9ac412a627304ad0631a3af31d86d8ac628). All CI [branches](https://gitlab.com/freecad/FreeCAD-CI/-/branches/all) and [pipelines](https://gitlab.com/freecad/FreeCAD-CI/-/pipelines?scope=branches).

**Comment by luzpaz** [comment]
@chennes looks like this PR is green light based on this recent commit by wandererfan: 058f88ec8bf4

**Pull request merged**